### ISSUE TITLE
New blog posts not being added to blogroll

### ISSUE THREAD

**Issue by Ordresser** [datenstrom/yellow#486]
I've used Datenstrom Yellow for about a year and like it a lot. 

Recently, I tried blogging a few blogs, and they seem to save (and actually do create new .md files in the content directory) but they don't get printed on the blogroll. Any ideas what's happening? It always worked before. I waited a few days and tried a few posts spaced out, because sometimes it used to take a day until the whole site was updated (although blogroll I think added new posts immediately), but nothing has happened with the new blogs.

**Comment by BsNoSi** [comment]
I presume that you have cleared your browser cache which sometimes can lead to this effect.

One thing often happens to me: "status". 
If set to "draft" or "private"page stays invisible. 
Removing the "status" solves this imediately.

**Comment by schulle4u** [comment]
Sounds like your blog posts are missing the page layout setting. Have you checked your `/content/shared/page-new-blog.md` template after updating Yellow? It should contain the right layout page setting.

**Comment by BsNoSi** [comment]
Possibly it would help to have a copy of the header of one of these files, that should look somehow like this:

```
---
title: 
TitleSlug:
description: 
tag: 
Layout: blog
published: 
modified: 
author: 
language: de 
Sidebar: sidebar
SocialtagsTwitterUser:
Image:  
ImageAlt: 
Status: draft 
---
```

**Comment by Ordresser** [comment]
OK, I found how to do a quick fix, but would still like to actually fix it.

In my page-new-blog.md file I do have this:

```
---
Title: Blog page
Published: @datetime
Author: @username
Layout: blog
Tag: Example
---
This is a new blog page.
```

But that doesn't appear when I click the plus to add a new blog post. When I do that on the front end, I get: 

---
Title: Page
---
This is a new page.

(Before this issue appeared, I used to get the full list of stuff to put there when I clicked the plus.)

If I cut and paste the proper header info and change the date to an appropriate one, it publishes/displays the post correctly on the front end.
